nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -140,11 +140,15 @@ def get_start_nodes_for_activation_path_tracing(nncf_graph: PTNNCFGraph) -> List
 
     @staticmethod
     def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> PTTargetPoint:
+        input_port_id: Optional[int] = port_id
```

Comment:
Why input_port_id is optional here and required as the method param

Reply:
Because input_port_id can be None for post operation hooks

Reply:
Please update the type hint for the port_id method param then

Reply:
Ok, got it